src/almanac/verdant-sundial/verdant-sundial.md

```diff
@@ -0,0 +1,40 @@
+# Verdant Sundial
+
+```{figure} ../assets/Sundial_2916_HDR.jpeg
+---
+alt: Sundial on a gravestone.
+---
+_The verdant sundial casts shadows of past and future intention observed in the present. (Image source: {cite:alp}`image_elfward_sundial`)._
+```
+
+> "The present is the ever moving shadow that divides yesterday from tomorrow. In that lies hope." <br>
+> \- Frank Lloyd Wright
+
+The software garden is intertwined in a story of time which is exhibited like a shadow on a moss-covered sundial.
+Time is observed indirectly through a kind of software archeology in past-tense material artifacts (files) and present-tense operation (procedures).
+These artifacts act as shadows of our prior creative intentions where procedures become the living present materialized promise of those intentions.
+"This brings an uncanny sense of living in a state where the future is a dimension of our present reality." {cite:p}`morton2013hyperobjects`
+The future is elusive but present in software artifacts, similar to rings of a tree that depict the hope of change.
```

Comment:
```suggestion
The future is elusive but present in software artifacts, similar to rings of a tree that depict the promise of continued stability or hope of change.
```

I suppose this suggestions depends on what you mean by "software artifacts" - if somethign to avoid, then hope of change words, but if something else, then a stable future is desirable